Sources/SnapshotTesting/Strategy/CaseIterable.swift

```diff
@@ -0,0 +1,22 @@
+
+extension Strategy where Snapshottable: CaseIterable, Format == String  {
+
+  public static func `func`<A>(into witness: Strategy<A, Format>) -> Strategy<(Snapshottable) -> A, Format> {
+
+    var strategy = Strategy<String, String>.lines.asyncPullback { (f: (Snapshottable) -> A) in
+      Snapshottable.allCases.map { input in
+        witness.snapshotToDiffable(f(input))
+          .map { (input, $0) }
+        }
+        .sequence()
+        .map { rows in
+          rows.map { "\"\($0)\",\"\($1)\"" }
```

Comment:
Could even do something like `"\($0) → \($1)"`.

Reply:
in this case I'm snapshotting as a CSV so that we can get this formatting when viewing the file on disk:

<img width="366" alt="image" src="https://user-images.githubusercontent.com/135203/48555370-01af8100-e8af-11e8-9f3a-3557f67588bb.png">

The reason I did this is cause I didn't want to have to worry about about nicely formatting columns. what do you think?